packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -194,6 +194,23 @@ function DataViews< Item >( {
 		}
 	}, [ hasPrimaryOrLockedFilters, isShowingFilter ] );
 
+	// Auto-sort timeline by eventField if defined
```

Comment:
@oandregal unsure about this part. If you're switching between views its possible the sort order won't be by date. How would you go about this?

Reply:
What if we get rid of the `eventField` and instead make `view.sort.field` required for the timeline layout? I can't think of any case where the sort field and the event field need to be different for timeline.